embc-app/ClientApp/src/app/home/home.component.scss

```diff
@@ -1,80 +1,90 @@
-@import "~src/styles/_defaults.scss";
+// @import "~src/styles/_defaults.scss";
 
-$container-width: auto; // 775px
+// $container-width: auto; // 775px
 
-// ==> cards
-.picture-card {
-  max-width: $card-width;
-  background-color: $card-body-bg;
-  padding-bottom: 48px;
-}
-
-// ==> home
+.card-img-top {
+  width: 100%;
+  height: 15vw;
+  object-fit:scale-down;
 
-.home-margins {
-  max-width: 1200px;
-  // padding-left: 30px;
-  // padding-right: 30px;
 }
+.card {
+  background-color: rgba(241, 244, 250, 1);
```

Comment:
Isn't there a SCSS variable you can use for the color here?

Reply:
Done